### PR TITLE
Fixed Project Title and Mission

### PR DESCRIPTION
**Pull request by b3ll**
Basically playing smash bros is way more fun than dealing with iOS development.

Also can confirm build passing 100%.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,191 +1,81 @@
...
+no u.
 
 ## Special Thanks
 
-Thanks to all [contributors](https://github.com/fastlane/fastlane/graphs/contributors) for extending and improving `fastlane`. Check out the project pages of the other tools for more sponsors and contributors.
+|   |                                          |
+|---|------------------------------------------|
+|🙃|[@b3ll](https://www.twitter.com/b3ll)      |
+|🎉|[@KrauseFx](https://www.twitter.com/b3ll)  |
+|💮|[@ayanonagon](https://www.twitter.com/b3ll)|
```
Comment (ayanonagon) [inline README.md]: ಠ_ಠ

**Comment by ayanonagon** [comment]
:white_flower:

**Comment by neonichu** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (neonichu) [inline README.md]: Always wondered why you three look so similar, finally we know...

**Comment by KrauseFx** [comment]
:+1:

**Comment by KrauseFx** [comment]
Tests are passing, code style is okay, I think it's ready to merge

**Comment by b3ll** [comment]
:+1: should merge

**Comment by neonichu** [comment]
still not merged...

**Comment by KrauseFx** [comment]
@neonichu is still missing in the README, can't merge without him

**Comment by b3ll** [comment]
@KrauseFx I added @neonichu and tests are still 100% passing, pls merge. thx in advances

**Comment by KrauseFx** [comment]
@b3ll Why is @neonichu a 🌮?

**Comment by b3ll** [comment]
closest thing to a nacho I could find

**Comment by neonichu** [comment]
that seems reasonable, plz merge :+1:

**Comment by jcampbell05** [comment]
Conflics now :(

**Comment by KrauseFx** [comment]
Project mission merge conflict :cry:

**Comment by b3ll** [comment]
crap, will fix 🙃

**Comment by KrauseFx** [comment]
@b3ll You don't take open source project serious enough, I've been waiting for a fix for weeks now. Will close as duplicate of https://github.com/fastlane/fastlane/commit/b0e927629b22875c153e2629fe88a9f7e051097d

**Comment by neonichu** [comment]
😭

**Comment by KrauseFx** [comment]
:sob: (bigger emoji)

**Comment by neonichu** [comment]
Always rejecting the good PRS...

**Pull request closed without merging**